### PR TITLE
Feat/init

### PR DESCRIPTION
**Pull request by DavidTrujillo123**

### CODE REVIEW

**Review by juanfcarrillo — COMMENTED** [review]
There's some issue, you forgot to add the @storyblok/astro to the package.json file. Check if you are ignoring this file

File: src/pages/colegio.astro [hunk 1]
```diff
@@ -13,16 +13,39 @@ import Layout from "../layouts/Layout.astro";
...
 import CondritaSection from "../components/sections/CondritaSection.astro";
 import CountDown from "../components/countDown/CountDown.astro";
+import Ejemplo from "../components/ejemplo.astro";
+import { storyblokEditable } from '@storyblok/astro'
+import StoryblokComponent from "@storyblok/astro/StoryblokComponent.astro";
+const { blok } = Astro.props
+
+
+
+import { useStoryblokApi } from '@storyblok/astro'
```
Comment (ferjozsot23) [inline src/pages/colegio.astro:23]: Try to not add plenty blank spaces

File: src/pages/colegio.astro [hunk 2]
```diff
@@ -13,16 +13,39 @@ import Layout from "../layouts/Layout.astro";
...
+
+const { data } = await storyblokApi.get("/cdn/stories/colegio", {
+  version: "draft",
+});
+const content = data.story.content;
+console.log(content);
+
+
+
+
```
Comment (ferjozsot23) [inline src/pages/colegio.astro:36]: Same here

File: src/components/ejemplo.astro
Comment (ferjozsot23) [inline src/components/ejemplo.astro:1]: We can keep the file names in English for consistency

**Review by ferjozsot23 — DISMISSED** [review]
Looks great! just some recommendation for good practices

**Review by juanfcarrillo — APPROVED** [review]
🚀

**Pull request merged**